test/any/parsing/GeckoParser-test.js

```diff
@@ -12,7 +12,7 @@ describe('GeckoParser', function() {
   });
 
   function parse(str, opt_production) {
-    var p = GeckoParser.create().parseString(str, opt_production);
+    var p = GeckoParser.create().parseString(str, null, opt_production);
```

Comment:
If we go with the factory+validate pattern, null here might need to be changed to 'test' or something.